examples/golang/bucket-notification-deletion/README.md

```diff
@@ -0,0 +1,11 @@
+# Introduction
+This directory contains Golang code example on how to create an SNS Topic on a Ceph RGW cluster.
+
+# Prerequisite
+Linux machine running an RGW Ceph cluster. Preferrably started with the ``OSD=1 MON=1 MDS=0 MGR=0 RGW=1 ../src/vstart.sh --debug --new `` command.  
+Go installed on the Linux machine.  
+
+## Workflow Procedure
```

Comment:
please explain about the sub-directory structure here

Reply:
I was thinking of making the notification code as an extension that can be used in the example code.
The notification-deletion folder contains the custom Rook code written and the notification-deletion-example folder has the example code showing how a notification can be deleted.
I intended to call the code in the notification-deletion example as a custom package in the notification-deletion-example code as per the info at https://golangbot.com/go-packages/

Reply:
ok. so you have to explain in the README what to do where;
"Navigate through your file system to where the Golang example code exists on your terminal"
does not explain to which directory should we navigate